conda_meta/full/meta.yaml

```diff
@@ -35,12 +34,10 @@ requirements:
     - psutil
     - Pillow
     - requests
-    - sigopt
     - prettytable
     - cryptography
```

Comment:
This `cryptography` should also be removed, right?

Reply:
No, full package require this component.

Reply:
okay, LGTM